BlobFeed.pushproj

```diff
@@ -0,0 +1,37 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTarget="PublishToFeed">
```

Comment:
Why not use the Arcade SDK for publishing to a blob feed?

Reply:
Because it needs to happen in a release where the source code isn't available. Arcade doesn't play well with releases.